themes/default/content/docs/get-started/_index.md

```diff
@@ -13,39 +13,62 @@ aliases:
+        srcdoc="<style>*{padding:0;margin:0;overflow:hidden}html,body{height:100%}img{position:absolute;width:100%;top:0;bottom:0;margin:auto}</style><a href=https://www.youtube.com/embed/6f8KF6UGN7g?autoplay=1><img src='/images/home/youtube-preview.svg' alt='Introduction to Pulumi: Modern Infrastructure as Code'></a>">
+    </iframe>
+</div>
+
+See [Cloud Providers]({{< relref "/docs/intro/cloud-providers" >}}) on how to use other supported clouds.
+
+## Additional resources
+
+The following sections are also useful when first learning how to use Pulumi:
 
 <div class="md:flex flex-row mt-6 mb-6">
-    <div class="w-full border-solid border-t-2 border-gray-200">
-        <h3 class="no-anchor pt-4"><i class="fas fa-laptop-code pr-2"></i>Local Development</h3>
-        <p>Deploy a simple application from your local machine to one of the cloud providers below.</p>
-        <div class="tiles flex-wrap mt-4">
-            <div class="pb-4 md:pr-4 md:w-1/2">
-                <a class="tile p-8" href="{{< relref "/docs/get-started/aws" >}}">
-                    <img class="h-10 mx-auto" src="/logos/tech/aws.svg" alt="AWS">
-                </a>
-            </div>
-            <div class="pb-4 md:w-1/2">
-                <a class="tile p-8" href="{{< relref "/docs/get-started/kubernetes" >}}">
-                    <img class="h-10 mx-auto" src="/logos/tech/k8s.svg" alt="Kubernetes">
-                </a>
-            </div>
-            <div class="pb-4 md:pr-4 md:w-1/2">
-                <a class="tile p-8" href="{{< relref "/docs/get-started/azure" >}}">
-                    <img class="h-10 mx-auto" src="/logos/tech/azure.svg" alt="Azure">
-                </a>
-            </div>
-            <div class="pb-4 md:w-1/2">
-                <a class="tile p-8" href="{{< relref "/docs/get-started/gcp" >}}">
-                    <img class="h-10 mx-auto" src="/logos/tech/gcp.svg" alt="Google Cloud">
-                </a>
-            </div>
-        </div>
-        <p>See <a href="{{< relref "/docs/intro/cloud-providers" >}}">Cloud Providers</a> for other clouds.</p>
+    <div class="md:w-1/2 border-solid border-t-2 border-gray-200">
+        <h3 class="no-anchor pt-4"><a href="{{< relref "/docs/intro/concepts" >}}"><i class="fas fa-file-alt pr-2"></i>Architecture & Concepts</a></h3>
+        <p>Get details on the Pulumi programming model and core concepts.</p>
+    </div>
+    <div class="md:w-1/2 md:ml-4 border-solid border-t-2 border-gray-200">
+        <h3 class="no-anchor pt-4"><a href="{{< relref "/docs/guides/adopting" >}}"><i class="fas fa-cloud pr-2"></i>Adopting Pulumi</a></h3>
+        <p>Learn how to support, migrate, or convert existing cloud infrastructure with Pulumi.</p>
     </div>
 </div>
+
 <div class="md:flex flex-row mt-6 mb-6">
     <div class="w-full border-solid border-t-2 border-gray-200">
         <h3 class="no-anchor pt-4"><i class="fas fa-users pr-2"></i>Live Workshops</h3>
```

Comment:
This section doesn't seem to have a link on the header, and instead has a hyper-linked word in the description. Should we be consistent between the three sections?

Reply:
Good catch. Yup, I'll fix.